library/keystone_user.py

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/python
```

Comment:
@masteinhauser because we added functionality to handle keystone roles without assigning a user for a use case in this playbook.   a PR was done to upstream and accepted,  but we need to put it in here until we pull that module into our forked ansible and cut a new release.